src/app/lib/config/services/punjabi.js

```diff
@@ -19,7 +20,7 @@ const service = {
   defaultCaptionOffscreenText: 'Caption, ',
   imageCopyrightOffscreenText: 'Image source, ',
   locale: `pa-IN`,
-  datetimeLocale: `pa-IN`.toLowerCase(),
+  datetimeLocale: `pa-in`,
```

Comment:
This should be kept as it was. Because `pa-in` is not a valid `datetimeLocale` for punjab india. `pa-IN` is valid however that would not work with moment. Thats why there is a `toLowerCase` call. Here is a [link](http://para.se/perldoc/DateTime/LocaleCatalog.html) to datetime locales.

Reply:
Makes sense. I have reverted